### ISSUE TITLE
cordova-plugin-code-push will install 1.5.0-beta instead of 1.5.1-beta on npm

### ISSUE THREAD

**Issue by zwacky** [Microsoft/cordova-plugin-code-push#71]
`ionic plugin add cordova-plugin-code-push` will install 1.5.0-beta. had to use `ionic plugin add cordova-plugin-code-push@1.5.1-beta` to get the latest version.

npm bump?

**Comment by lostintangent** [comment]
Hmm, I thought we added the "latest" tag to 1.5.1 when we released it, but maybe we didn't. I'll fix this as soon as I get him in a little bit. Sorry dude!

**Comment by lostintangent** [comment]
I just tried running `cordova plugin add cordova-plugin-code-push` and `ionic plugin add cordoba-plugin-code-push` (even though they're the same, I wanted to do a sanity test) on new projects, and in each case, I got the `1.5.1-beta` version of the CodePush plugin. I also confirmed via `npm view cordova-plugin-code-push dist-tags` that `1.5.1-beta` is in fact tagged as the `latest` released (which is what you get when you NPM install something with an unqualified version).

I'm wondering if maybe NPM was re-using a cached version of the plugin to satisfy your install request, and only after you specified the version, did it pull it down. I'll update our docs to specify passing `@latest` when doing the NPM install, since that tends to be a good fail safe way of getting around this issue.
